ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1964,4 +1964,37 @@ def test_tensorflow_sinh(
         fn_tree=fn_tree,
         on_device=on_device,
         x=x[0],
-    )
\ No newline at end of file
+    )
+
+
+# zeta
+@handle_frontend_test(
+    fn_tree="tensorflow.math.zeta",
+    dtype_and_x=helpers.dtype_and_values(
```

Comment:
You'll need to add an argument here for the alias raw_ops function. Here you can read the documentation on how we are testing aliases https://lets-unify.ai/ivy/overview/deep_dive/ivy_frontends_tests.html#alias-functions.